api/openapi-spec/swagger.json

```diff
@@ -10322,6 +10326,10 @@
      "operatorVersion": {
       "type": "string"
      },
+     "outdatedVMIWorkloads": {
```

Comment:
Really minor: How about either `outdatedWorkloads` or `outdatedVirtualmachineInstanceWorkloads`? I know it is long but we don't use abbreviations in the API for our our components afaik.

Reply:
yep, that makes sense to me.

Reply:
fixed